x-pack/plugins/security_solution/common/api/timeline/persist_note/persist_note_route.ts

```diff
@@ -19,3 +19,22 @@ export const persistNoteSchema = runtimeTypes.intersection([
     version: unionWithNullType(runtimeTypes.string),
   }),
 ]);
+
+export const persistNoteWithoutRefSchema = runtimeTypes.intersection([
+  runtimeTypes.type({
+    note: BareNoteWithoutExternalRefsSchema,
+  }),
+  runtimeTypes.partial({
```

Comment:
Can you also update the openapi schema here https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/api/timeline/persist_note/persist_note_route_schema.yaml ? Thanks!